clayui.com/src/components/Editor/index.js

```diff
@@ -55,16 +55,30 @@ const Editor = ({
 	}
 
 	function useStickyState(defaultValue, key) {
+		const inBrowser = typeof window !== 'undefined';
+
+		const localStorage = {
+			getItem(key) {
+				return inBrowser ? window.localStorage.getItem(key) : null;
+			},
+
+			setItem(key, value) {
+				if (inBrowser) {
+					window.localStorage.setItem(key, value);
+				}
+			},
+		};
+
```

Comment:
You could move all of this outside of `useStickyState` so as not to redefine it each time the hook is used.

Reply:
And actually, now that I look at the broader context I see the entire definition of the `useStickyState` hook itself could/should be moved outside of the component.

Reply:
I agree @wincent it can be moved outside, I'll play around with it soon. Thanks for taking the time to look at it in a broader context 🥇